nondex-common/src/main/java/edu/illinois/nondex/common/ConfigurationDefaults.java

```diff
@@ -60,6 +60,9 @@ a copy of this software and associated documentation files (the
     public static final String DEFAULT_END_STR = "9223372036854775807";
     public static final long DEFAULT_END = new Long(ConfigurationDefaults.DEFAULT_END_STR);
 
+    public static final String PROPERTY_LOG = "logFile";
```

Comment:
All nondex properties start with nondex*.
you should name the property nondexSystematicLogFile.
There may be other plugins that use logFile as an argument... then you won't be able to use nondex at the same time with that plugin.